lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -0,0 +1,896 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import io
```

Comment:
It's hard to review the code change, when a file is renamed. It's good to have rename in a separated commit, and then update. It doesn't need to operate the commits too much. Please use "git rebase -i origin/main" and add "b" before this commit. It can stop and add a new commit. In that commit, you can rename files like this, then commit it and continue with "git rebase --continue". There may be conflict, but you can accept all coming changes, because there is no code change in last step.

Reply:
But the PR will still show it as a rename. Will you then review commits separately instead?

I will try to this.

Reply:
There is a view to see commit one by one. It would be better.

Reply:
Done. The last commit now has only the rename and the import fixes because of the rename.